src/client/component.js

```diff
@@ -3,8 +3,7 @@ const walkVars = require('./visitors/vars');
 const walkNode = require('./visitors/node');
 const utils = require('./utils');
 
-require('__IDYLL_SYNTAX_HIGHLIGHT__');
-let results = require('__IDYLL_AST__');
+// require('__IDYLL_SYNTAX_HIGHLIGHT__');
```

Comment:
This executes some code that makes only certain languages available to the syntax highlighter to avoid bloating the bundle. Maybe we could just require the entire lib during SSR since we don't need to worry about filesize there.